datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -1039,7 +1039,7 @@ CREATE TABLE positions(
   primary key (party_id, market_id, vega_time)
 );
 
-select create_hypertable('positions', 'vega_time', chunk_time_interval => INTERVAL '1 day');
+select create_hypertable('positions', 'vega_time', chunk_time_interval => INTERVAL '1 day', if_not_exists => true);
```

Comment:
If we're putting `if not exists` in here I guess we should also have them in `CREATE TABLE positions` too.

Reply:
The `goose Down` step has a `DROP TABLE positions cascade` statement, so if such a table were to exist, I'd assume something is wrong, and I'd expect to see an error in that case

Reply:
Yea I agree, I think the existence checks will just hide errors that we should just fix.